server/datastore/mysql/activities_test.go

```diff
@@ -178,18 +198,32 @@ func testListActivitiesStreamed(t *testing.T, ds *Datastore) {
 	_, err := ds.NewUser(context.Background(), u)
 	require.Nil(t, err)
 
-	require.NoError(t, ds.NewActivity(context.Background(), u, dummyActivity{
-		name:    "test1",
-		details: map[string]interface{}{"detail": 1, "sometext": "aaa"},
-	}))
-	require.NoError(t, ds.NewActivity(context.Background(), u, dummyActivity{
-		name:    "test2",
-		details: map[string]interface{}{"detail": 2},
-	}))
-	require.NoError(t, ds.NewActivity(context.Background(), u, dummyActivity{
-		name:    "test3",
-		details: map[string]interface{}{"detail": 3},
-	}))
+	timestamp := time.Now()
+	ctx := context.WithValue(context.Background(), fleet.ActivityWebhookContextKey, true)
+	require.NoError(
+		t, ds.NewActivity(
+			ctx, u, dummyActivity{
+				name:    "test1",
+				details: map[string]interface{}{"detail": 1, "sometext": "aaa"},
+			}, nil, timestamp,
+		),
+	)
+	require.NoError(
+		t, ds.NewActivity(
+			ctx, u, dummyActivity{
+				name:    "test2",
+				details: map[string]interface{}{"detail": 2},
+			}, nil, timestamp,
+		),
+	)
+	require.NoError(
+		t, ds.NewActivity(
+			ctx, u, dummyActivity{
+				name:    "test3",
+				details: map[string]interface{}{"detail": 3},
+			}, nil, timestamp,
+		),
+	)
```

Comment:
nit: is it worth adding a `require.Error` if the wrong or no context is passed to ds.NewActivity?